src/components/RadioButtonGroup/RadioButtonGroup.tsx

```diff
@@ -40,34 +47,61 @@ export interface RadioButtonGroupProps extends HTMLProps<HTMLDivElement> {
    * The ref to the HTML Dom element
    */
   ref?: Ref<HTMLDivElement>;
+  /**
+   * Warning or error message, displayed in a tooltip.
+   */
+  validationHint?: string;
+  /**
+   * Triggers error styles on the component. Important for accessibility.
+   */
+  invalid?: boolean;
 }
 
+const inlineMessageStyles = ({ theme }: StyleProps) => css`
+  margin-top: ${theme.spacings.mega};
+`;
+
+const StyledInlineMessage = styled(InlineMessage)<InlineMessageProps>(
+  inlineMessageStyles,
+);
+
 const RadioButtonGroupComponent = (
   {
     options,
     onChange,
     value: activeValue,
     name: customName,
     label,
+    invalid,
+    validationHint,
     ...props
   }: RadioButtonGroupProps,
   ref: RadioButtonGroupProps['ref'],
 ) => {
   const name = customName || uniqueId('radio-button-group_');
   return (
-    <div role="group" aria-label={label} ref={ref} {...props}>
-      {options &&
-        options.map(({ children, value, className, ...rest }) => (
-          <div key={value && value.toString()} className={className}>
-            <RadioButton
-              {...{ ...rest, value, name, onChange }}
-              checked={value === activeValue}
-            >
-              {children}
-            </RadioButton>
-          </div>
-        ))}
-    </div>
+    <Label htmlFor={name}>
+      {label && <span>{label}</span>}
```

Comment:
The Label component / `label` HTML element is meant for single inputs only. For field groups, you should use the `fieldset` and `legend` HTML elements (see this [implementation example](https://dequeuniversity.com/library/aria/radio-and-radio-group)).

You can still use the Label component for the styles though: `<Label as="legend">`.